internal/users/users_embed.go

```diff
@@ -5,15 +5,19 @@ import (
 )
 
 var (
+	// EmptyExample ...
```

Comment:
Let's complete these.

Reply:
I was actually thinking about bringing the internal/users package inside the internal/cli/data pkg and then make these private, as they're pretty self explanatory, wdyt?

Reply:
I completed the comments for now tho: [`014023f` (#669)](https://github.com/auth0/auth0-cli/pull/669/commits/014023f9e68a72e4151559f8d9fdc8ea942dd3bc).

Reply:
> I was actually thinking about bringing the internal/users package inside the internal/cli/data pkg and then make these private, as they're pretty self explanatory, wdyt?

Yeah, no need to have them be public.